formula/src/main/java/com/instacart/formula/Inspector.kt

```diff
@@ -61,6 +62,11 @@ interface Inspector {
      */
     fun onActionFinished(formulaType: KClass<*>, action: DeferredAction<*>) = Unit
 
+    /**
+     * Called when transition [Effects] are executed
+     */
+    fun onEffectExecuted() = Unit
```

Comment:
This callback would probably be much more useful if we passed the effect here, right? Wouldn't it typically be an anonymous class hinting at where the execution point is?

Reply:
I've removed this for now, will add it in another PR